source/extensions/common/sqlutils/sqlutils.cc

```diff
@@ -23,7 +32,7 @@ bool SQLUtils::setMetadata(const std::string& query, ProtobufWkt::Struct& metada
     // Get names of accessed tables.
     result.getStatement(i)->tablesAccessed(table_access_map);
     for (auto& it : table_access_map) {
-      auto& operations = *fields[it.first].mutable_list_value();
+      auto& operations = *fields[it.first + database].mutable_list_value();
```

Comment:
you sure this doesn't break mysql? or did you fix up the sql parser for this?

Reply:
No it does not break mysql. I double checked that. For mysql ``database`` is empty string.